test/e2e/azure_test.go

```diff
@@ -294,6 +294,67 @@ var _ = Describe("Workload cluster creation", func() {
 		})
 	})
 
+	When("Creating a highly available cluster [REQUIRED] with Azure CNI v1", Label("Azure CNI v1"), func() {
+		It("can create 3 control-plane nodes and 2 Linux worker nodes", func() {
+			clusterName = getClusterName(clusterNamePrefix, "azcni-v1")
+
+			clusterctl.ApplyClusterTemplateAndWait(ctx, createApplyClusterTemplateInput(
+				specName,
+				withAzureCNIv1Manifest(e2eConfig.GetVariable(AzureCNIv1Manifest)), // AzureCNIManifest is set
+				withFlavor("azure-cni-v1"),
+				withNamespace(namespace.Name),
+				withClusterName(clusterName),
+				withControlPlaneMachineCount(3),
+				withWorkerMachineCount(2),
+				withControlPlaneInterval(specName, "wait-control-plane-ha"),
+				withControlPlaneWaiters(clusterctl.ControlPlaneWaiters{
+					WaitForControlPlaneInitialized: EnsureControlPlaneInitialized,
+				}),
+				withPostMachinesProvisioned(func() {
+					EnsureDaemonsets(ctx, func() DaemonsetsSpecInput {
+						return DaemonsetsSpecInput{
+							BootstrapClusterProxy: bootstrapClusterProxy,
+							Namespace:             namespace,
+							ClusterName:           clusterName,
+						}
+					})
+				}),
+			), result)
+
+			By("can expect VM extensions are present on the node", func() {
+				AzureVMExtensionsSpec(ctx, func() AzureVMExtensionsSpecInput {
+					return AzureVMExtensionsSpecInput{
+						BootstrapClusterProxy: bootstrapClusterProxy,
+						Namespace:             namespace,
+						ClusterName:           clusterName,
+					}
+				})
+			})
+
+			By("can validate failure domains", func() {
+				AzureFailureDomainsSpec(ctx, func() AzureFailureDomainsSpecInput {
+					return AzureFailureDomainsSpecInput{
+						BootstrapClusterProxy: bootstrapClusterProxy,
+						Cluster:               result.Cluster,
+						Namespace:             namespace,
+						ClusterName:           clusterName,
+					}
+				})
+			})
+
+			By("can create an accessible load balancer", func() {
```

Comment:
Based on the HA test immediately preceding this, just double checking that the following blocks aren't relevant and practical in this scenario?
- `Validating network policies`
- `Creating an accessible load balancer for windows`

Reply:
Thanks for bringing it up! Both of the them are relevant and practical scenarios which I aim to add in further iterations of working with Azure CNI v1. 
But a good reminder to myself to update the tasks in the parent epic on investigating and updating the tests with network policies and e2e tests for windows nodes.